### PR TITLE
Draggable overlay window

### PR DESCRIPTION
**Pull request by scottwittenburg**

### CODE REVIEW

File: src/React/Containers/OverlayWindow/index.js [hunk 1]
```diff
@@ -0,0 +1,414 @@
...
+        top: this.props.marginSize,
+        right: this.props.marginSize,
+        left: this.props.marginSize,
+        height: this.props.titleBarHeight,
+        lineHeight: `${this.props.titleBarHeight}px`,
+      },
+    };
+
+    // Configure the hot corner divs
+    const off = this.props.hotCornerExtra;
```
Comment (aronhelser) [inline src/React/Containers/OverlayWindow/index.js]: kill this line, use this.props.hotCornerExtra directly

**Comment by aronhelser** [comment]
LGTM, but I don't know the context...

File: src/React/Containers/OverlayWindow/index.js (hunk 1 above)
Comment (scottwittenburg) [inline src/React/Containers/OverlayWindow/index.js]: I'm ok with that.  I guess you figured out the reason for the short-lived variable: four lines in the jsx section below got a little unwieldy.

Thanks for taking a look :-)  I'll show you the context next time we hangout.

**Comment by scottwittenburg** [comment]
@jourdain, mind taking a look when you get a chance?

**Comment by jourdain** [comment]
LGTM

**Pull request merged**